certora/specs/Liveness.spec

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+import "Reverts.spec";
```

Comment:
I agree that it makes it harder to know where we are. This is also because splitting was needed for performance. Not sure yet what's the best solution to that is, maybe have a dependence graph somewhere could help too